source/run-command/run-command.rst

```diff
@@ -0,0 +1,185 @@
+       *
+       * @ see https://github.com/mongodb/specifications/blob/master/source/client-side-operations-timeout/client-side-operations-timeout.rst
+       */
+      timeoutMS?: number;
+    }
+
+Deviations
+^^^^^^^^^^
+
+* Drivers MAY rename their API to match the language syntax.
+
+RunCommand implementation details
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+RunCommand provides a way to access MongoDB server commands directly without requiring a driver to implement a bespoke helper.
+The API is intended to take a document from a user and apply a number of common driver internal concerns before forwarding the command to a server.
+In general, a driver SHOULD avoid inspecting or modifying the user's command document in an effort to remain forward compatible with any potential server commands.
+
+Drivers that have historically modified user input SHOULD strive to instead clone the input such that appended fields do not affect the user's input.
+
+OP_MSG
+""""""
+
+The ``$db`` global command argument MUST be sent on the command sent to the server and it MUST correspond to the database name the runCommand API was invoked on.
+
+* See OP_MSG's section on `Global Command Arguments <https://github.com/mongodb/specifications/blob/master/source/message/OP_MSG.rst#global-command-arguments>`_
+
+ReadPreference
+""""""""""""""
+
+A driver's runCommand implementation MUST assume all commands are read operations.
+To facilitate selecting the server the command should be sent to a driver's runCommand MUST accept an optional Read Preference option.
+
+* See Server Selection's section on `Use of read preferences with commands <https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#use-of-read-preferences-with-commands>`_
+
+If the provided Read Preference is NOT primary, the command sent MUST include the ``$readPreference`` global command argument.
+
+* See OP_MSG's section on `Global Command Arguments <https://github.com/mongodb/specifications/blob/master/source/message/OP_MSG.rst#global-command-arguments>`_
+
+Driver Sessions
+"""""""""""""""
+
+A driver's runCommand API MUST provide an optional session option to support explicit sessions and transactions.
+If a session is not provided the driver MUST attach an implicit session if the selected server supports sessions.
+
+Every ClientSession has a corresponding Logical Session ID representing the server side session ID.
+The Logical Session ID MUST be included under ``lsid`` in the command sent to the server without modifying user input.
+
+* See Driver Sessions' section on `Sending the session ID to the server on all commands <https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst#sending-the-session-id-to-the-server-on-all-commands>`_
+
+Transactions
+""""""""""""
+
+If RunCommand is used within a Transaction the read preference MUST be sourced from the Transaction's options.
```

Comment:
```suggestion
If runCommand is used within a transaction the read preference MUST be sourced from the transaction's options.
```

"runCommand" seemed consistent with prior text in the spec. If you prefer "RunCommand", I'd suggest going through and changing everything to a single form.

Transaction isn't a proper noun, so we can lowercase that.

"read preference" is fine as-is since you're not referring to the special type.

Alternatively, "transaction's options" can refer to the special type: [TransactionOptions](https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#transactionoptions).